specification/semantic_conventions.md

```diff
@@ -162,24 +165,19 @@ the top of the stack provided first and the bottom of the stack provided last.
 The format of each stack trace line is:
 
 ```
-        at <namespace>.<function>(<file>:<lineno>)
-        at <namespace>.<function>(<file>:<lineno> <col>)
-        at <namespace>.<function>(<file>:<lineno>:<lineno> <col>:<col>)
+  at <function>(<file>:<lineno>)
+  at <function>(<file>:<lineno>:<col>)
 ```
 
-- OPTIONAL leading whitespace
+- OPTIONAL leading whitespace (tabs or spaces)
 - OPTIONAL "at "
-- `<namespace>` - REQUIRED. Fully qualified namespace of the function call. For object-oriented
-  languages this SHOULD be the module/package and class name. If the function being executed is globally 
-  scoped this MUST be set to `global`. If class is unknown, the value MUST be set to `unknown`.
-- `<function>` - Name of the function or class method being invoked
+- `<function>` - REQUIRED. Fully qualified name, i.e. containing the namespace, module and/or class name if applicable, of the function or method being invoked.
 - literal `(`
 - `<file>` - REQUIRED. The name of the source code file (including filename extension). If the function being
   executed is OS native, an implementation-specific indication SHOULD be used. If the source module
-  cannot be determined (eg. when symbols have been excluded), the value MUST be `unknown`
-- literal `:` - REQUIRED if `<line>` is known, OPTIONAL otherwise.
-- `<lineno>` or `<lineno>:<lineno>` - RECOMMENDED - The line of source code that corresponds with the function invocation point. If the runtime 
-provides a range of lines, the second `lineno` can be provided after a colon `:`.
-- ` <col>` or ` <col>:<col>` - OPTIONAL - If the runtime provides a column or column range, it MAY be provided
-after the `lineno`, separated by a space.
+  cannot be determined (eg. when symbols have been excluded), the value MUST be `unknown`. The file MAY contain additional `:` characters.
```

Comment:
Just curious, from the description I see:

* Allow : characters in file names: Node.js internal library file names may have : in them.

How are those named on Windows? (`:` is not accepted character in Windows file names)

Reply:
On Windows it still contains `:`, these special names don't point to files, but internal Node.js modules. For example `node:events:397` points to https://github.com/nodejs/node/blob/master/lib/events.js#L397